dsc_lib/src/dscresources/dscresource.rs

```diff
@@ -245,12 +245,8 @@ pub fn get_diff(expected: &Value, actual: &Value) -> Vec<String> {
     }
```

Comment:
Instead of the code needed to explicitly check `_exist`, simply add it to `expected` and `actual` if it doesn't already contain it with default value `true` and the of the validation works as-is